src/DurableTask.SqlServer.AzureFunctions/SqlMetricsProvider.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace DurableTask.SqlServer.AzureFunctions
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    class SqlMetricsProvider
+    {
+        readonly SqlOrchestrationService service;
+        readonly int? previousWorkerCount;
+
+        public SqlMetricsProvider(SqlOrchestrationService service, int? previousWorkerCount = null)
```

Comment:
Adding a note that having `int? previousWorkerCount = null` isn't ideal and will change to get the current worker count.

Reply:
I realized that `previousWorkerCount` is only used to emit a trace [here](https://github.com/microsoft/durabletask-mssql/blob/1b5ccddd485a9b1da67d54076f77288c1725b5f2/src/DurableTask.SqlServer/SqlOrchestrationService.cs#L888C1-L891) so it seems fine to leave it as a nullable argument.

```
if (currentReplicaCount != null && currentReplicaCount != recommendedReplicaCount)
{
    this.traceHelper.ReplicaCountChangeRecommended(currentReplicaCount.Value, recommendedReplicaCount);
}
```